frontend/src/pages/AccountSend.vue

```diff
@@ -469,16 +477,60 @@ function useSendForm() {
       // This does not account for gas fees, but this gets us close enough and we delegate that to the wallet
       await getTokenBalances();
       const { address: tokenAddress, decimals } = token.value;
-      const tokenAmount = parseUnits(humanAmount.value, decimals);
+
+      const currentBalance = balances.value[tokenAddress];
+      const sendingNativeToken = tokenAddress === NATIVE_TOKEN.value.address;
+      let tokenAmount = parseUnits(humanAmount.value, decimals);
+      // Needed for transactions that are going to sendMax.
+      let preCalculatedGasPrice;
+      // Refresh the tokenAmount if the sendMax flag is set.
+      if (sendMax.value) {
+        if (sendingNativeToken) {
+          const [_getGasPrice, _toAddress, _estimatedNativeSendGasLimit] = await Promise.all([
+            provider.value!.getGasPrice(),
+            toAddress(recipientId.value, provider.value!),
+            estimateNativeSendGasLimit(),
+          ]);
+          // Add a 5% buffer to the gasPrice to avoid: "max fee per gas less than block base fee"
```

Comment:
Lots of comments seem L1-specific, such as "max fee per gas less than block base fee" and (paraphrased) "sending to a new address costs 25k extra gas". Maybe worth tweaking them

Reply:
Comments tweaked!